src/components/Main.ui.js

```diff
@@ -903,4 +903,10 @@ export default class Main extends Component<Props> {
     Actions.pop()
     return true
   }
+  checkAndShowGetCryptoModal = (props: any) => {
```

Comment:
This function seems unnecessary. Why not just add the sweepPrivateKey check to the original this.props.checkAndShow.. function? Then pass props.data to it